composer.json

```diff
@@ -23,7 +23,7 @@
         "ext-bcmath": "*",
         "ext-json": "*",
         "ext-openssl": "*",
-        "enjin/platform-core": "*",
+        "enjin/platform-core": "dev-feature/pla-1081/authorizable-type-fields",
```

Comment:
a mistake?

Reply:
No, was to make sure the tests passed 😄  Will revert before merging.